lib/content/getting-started.md

```diff
@@ -372,21 +362,20 @@ Objection ORM is an impressive SQL query-builder with a fantastic community, bui
 
 So, let's pull in our [Objection flavor](https://github.com/hapipal/boilerplate/commit/objection).
 
-If you used the hpal CLI to start your project as described above, just run:
+If you used the hpal CLI to start your project as described above, run:
 
 ```sh
 git cherry-pick objection
-npm install
```

Comment:
Is there an instruction elsewhere to install the new packages (objection, knex, etc.)?

Reply:
Ay, yea, it's a bunch of lines down, after explaining what all's happening with this command: https://github.com/hapipal/hapipal.com/pull/52/files#diff-663e1a280a7fe30ead22732740aa3d215335bf0c66967a35595d2bf153ae2108L435

Does that read alright to you? I was kept that as the only place where we instruct to install since the instructions right after this allude to resolving merge conflicts in package.json. But I could see lifting those up, so it reads something like: 

```diff
Expect to resolve small merge conflicts when pulling flavors in, typically just in `package.json`  and `server/manifest.js` having to do with overlapping dependencies in HEAD and the flavor.

+ Now let's check everything's still working:
+ 
+ ```sh
+ # bring in our new dependencies
+ npm install
+ npm start
+ ```
```

Do you prefer one over the other?

Reply:
Sounds like it's spoken to, all good to leave it as-is 👍